controllers/k8ssandra/k8ssandracluster_controller.go

```diff
@@ -73,6 +74,7 @@ type K8ssandraClusterReconciler struct {
 // +kubebuilder:rbac:groups=reaper.k8ssandra.io,namespace="k8ssandra",resources=reapers,verbs=get;list;watch;create;update;patch;delete
 // +kubebuilder:rbac:groups=core,namespace="k8ssandra",resources=pods;secrets,verbs=get;list;watch
 // +kubebuilder:rbac:groups=core,namespace="k8ssandra",resources=endpoints,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=monitoring.coreos.com,resources=servicemonitors,verbs=get;list;watch;create;update;patch;delete
```

Comment:
Missing `namespace="k8ssandra",` ?

Reply:
Since we decided to make this single namespace, we can indeed re-scope the permissions down to the k8ssandra namespace. Done.

Reply:
> Just so you are aware, the operator does support watching multiple namespaces, i.e., cluster-scoped

Thanks for the info. 

> Just so you are aware, the operator does support watching multiple namespaces, i.e., cluster-scoped. The changes to use ClusterRoles and ClusterRoleBindings is done in kustomize. I think the Helm chart needs to be updated to support it.

Kustomize changes already made (via `make manifests`), but what helm chart are we talking about here?

Reply:
Sorry, I am referring to https://github.com/k8ssandra/k8ssandra/tree/main/charts/k8ssandra-operator.

Reply:
Oh, interesting. I wasn't aware that this even existed. Thanks for bringing it to my attention. I will raise a separate ticket for it.